client/src/components/Review.jsx

```diff
@@ -1,11 +1,31 @@
 import React from 'react';
+import HostResponse from './HostResponse.jsx';
+
+const Review = (props) => {
+
+  const hasHostResponse = props.review.has_host_response; // 0 false, 1 true
+
+  const basicRender = (
+    <div>
+    <div className="review_name"><b>{props.review.first_name}</b></div>
```

Comment:
Could add some indentation between the open and close div for the nested elements so it's easier to visualize the html structure